crates/matrix-sdk-ui/src/spaces/mod.rs

```diff
@@ -389,7 +389,7 @@ impl SpaceService {
         space_state.graph = new_graph;
     }
 
-    async fn joined_spaces_for(client: &Client) -> (Vec<SpaceRoom>, SpaceGraph) {
+    async fn build_space_state(client: &Client) -> (Vec<SpaceRoom>, SpaceGraph) {
```

Comment:
Nitpicking: Since we are already on a type named `SpaceService`, I would name this method `build_state` directly.